.azure-pipelines/pipelines.yml

```diff
@@ -89,6 +89,27 @@ jobs:
         parameters:
           ciTarget: $(CI_TARGET)
 
+  - job: coverage
+    displayName: "Linux-x64 coverage"
+    dependsOn: ["format"]
+    timeoutInMinutes: 360
+    pool: "x64-large"
+    steps:
+      - template: bazel.yml
+        parameters:
+          managedAgent: false
+          ciTarget: bazel.coverage
+          rbe: ""
+          bazelBuildExtraOptions: "--test_env=ENVOY_IP_TEST_VERSIONS=v4only --curses=no"
+
+      - script: ci/run_envoy_docker.sh 'ci/upload_gcs_artifact.sh /build/envoy/generated/coverage coverage'
```

Comment:
Should the ${ENVOY_COVERAGE_DIR} in ci/build_setup.sh be removed?

Reply:
No, that's where coverage report generated to in `test/run_envoy_bazel_coverage.sh`. I hard coded here because in our docker run it is always this path.